modules/vba_documents/app/controllers/vba_documents/metadata_controller.rb

```diff
@@ -75,9 +77,9 @@ def healthcheck
       end
```

Comment:
You may want to wrap L72 to L76 in a `begin... rescue` block so that if something going wrong with the Slack notification, it doesn't affect the response returned by the healthcheck endpoint.

Reply:
Yea, I was thinking of doing that also when I wrote that code, thanks